StreamChatUITestsAppUITests/Robots/UserRobot.swift

```diff
@@ -159,5 +159,42 @@ final class UserRobot: Robot {
         app.back()
         return self
     }
+}
+
+// MARK: Debug menu
+
+extension UserRobot {
+
+    @discardableResult
+    func tapOnDebugMenu() -> Self {
+        MessageListPage.NavigationBar.debugMenu.tap()
+        return self
+    }
+
+    @discardableResult
+    func addParticipant() -> Self {
+        app.alerts["Select an action"].buttons["Add member"].firstMatch.tap()
+        app.textFields["debug_alert_textfield"].firstMatch.obtainKeyboardFocus().typeText("leia_organa")
```

Comment:
Should it always be `leia_organa`? An argument with a default value should work good I think.

Reply:
Done :)